sorc/build_all.sh

```diff
@@ -129,7 +129,7 @@ build_opts["ww3prepost"]="${_verbose_opt} ${_build_ufs_opt}"
 
 # Optional DA builds
 if [[ "${_build_ufsda}" == "YES" ]]; then
-   if [[ "${MACHINE_ID}" != "orion" && "${MACHINE_ID}" != "hera" ]]; then
+   if [[ "${MACHINE_ID}" != "orion" && "${MACHINE_ID}" != "hera.intel" && "${MACHINE_ID}" != "hercules" ]]; then
```

Comment:
@RussTreadon-NOAA 
The UFSWM has removed the `COMPILER` add-on to the `MACHINE_ID` and we will be removing it from `gfs-utils` as well soon.
The GSI, and other places will also be updated soon. 
FYI.

Reply:
Glad to hear this.  It's odd to have `MACHINE_ID` represent a machine + compiler.